sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -140,6 +140,18 @@ class FileStreamSourceSuite extends FileStreamSourceTest with SharedSQLContext {
 
   import testImplicits._
 
+  private def withSchemaInference(f: => Unit): Unit = {
+    withSQLConf(("spark.sql.streaming.allowSchemaInference", "true")) { f }
```

Comment:
Why do you use `{ f }` not `(f)`? Also, I've seen `"spark.sql.streaming.allowSchemaInference" -> "true"` used in the past.